mass_mailing_custom_unsubscribe/static/src/js/contact.tour.js

```diff
@@ -78,18 +75,20 @@ odoo.define("mass_mailing_custom_unsubscribe.contact_tour", function (require) {
             },
             {
                 content: "Subscribe again to list 0",
-                // eslint-disable-next-line no-multi-str
+                /* eslint-disable no-multi-str */
```

Comment:
Idk what is best... ignore no-multi-str or ignore max-length ... :/ for me, ignore "max-length" to better readability sounds better.

Reply:
In this case Eslint shows the warning (no-multi-str) because of the way the text is divided into several lines. If I ignore max-length it will still show the warning (no-multi-str)

Reply:
But the idea is to change the warning and join the multi line in one.